### PR TITLE
KevS-SK1757-Creation-SSL-DV-for-WebHosting

### PR DESCRIPTION
**Pull request by Kazarus4**
FRFR done.

Need tech review before peer rv.

I'll post a comment when tech rv has already done.

### CODE REVIEW

**Comment by Kazarus4** [comment]
Tech review done. 

Need peer rv before FRCA duplication and translations for all subs.

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+title: "Hébergement web - Activer un certificat SSL Sectigo DV"
+excerpt: "Découvrez comment activer un certificat SSL Sectigo DV sur votre hébergement Web OVHcloud"
+updated: 2024-10-18
+---
+
+## Objectif
+
+Les certificats Secure Socket Layer (SSL) permettent de chiffrer les échanges effectués depuis ou vers votre site web. Cela évite qu'une personne ou un robot malveillant ne vienne « écouter » clairement les requêtes envoyées ou émises avec votre site web.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
Les certificats Secure Socket Layer (SSL) permettent de chiffrer les échanges effectués depuis ou vers votre site web. Cela évite qu'une personne ou un robot malveillant ne vienne « écouter » clairement les requêtes émises depuis votre site web.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+OVHcloud propose plusieurs types de certificats SSL sur nos offres d'[hébergement mutualisé OVHcloud](/links/web/hosting). Ils sont présentés dans notre guide « [Hébergement web - Gérer un certificat SSL](/pages/web_cloud/web_hosting/ssl_on_webhosting) ». Les certificats SSL sont incontournables pour la sécurité de votre site web.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
Nous proposons plusieurs types de certificats SSL sur nos offres d'[hébergement mutualisé OVHcloud](/links/web/hosting). Ils sont présentés dans notre guide « [Hébergement web - Gérer un certificat SSL](/pages/web_cloud/web_hosting/ssl_on_webhosting) ». Les certificats SSL sont incontournables pour la sécurité de votre site web.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+Les niveaux de chiffrement SSL sont identiques entre ces trois types de certificat.
+
+La principale différence réside dans le niveau de vérifications qui sera réalisé par l'Autorité de Certification (AC) qui délivre le certificat SSL et atteste de son authenticité.
+
+Pour les hébergements mutualisés OVHcloud, l'autorité de certification délivrant les certificats SSL DV est [Sectigo](https://sectigostore.com){.external}.
+
+> [!warning]
+>
+> Sachez qu'une fois la commande initiée et transmise auprès de notre fournisseur de certificats/autorité de certification Sectigo, **aucun remboursement ne sera possible**.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> Une fois la commande du certificat réalisée et transmise à notre fournisseur de certificats/autorité de certification Sectigo, **aucun remboursement de commande n'est possible**.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+- Être connecté à votre [espace client OVHcloud](/links/manager).
+- Commander ou disposer d'un [hébergement mutualisé OVHcloud](/links/web/hosting) sur lequel aucun certificat SSL n'est déjà installé.
+- Commander ou disposer d'un [nom de domaine](/links/web/domains) et disposer des droits exclusifs sur son utilisation. Le nom de domaine ne doit pas déjà être lié à un certificat SSL.
+
+## En pratique
+
+> [!warning]
+>
+> Les certificats SSL Sectigo DV proposés chez OVHcloud ne sont valables que pour l'un des deux cas suivants sur votre hébergement web :
+> - un seul nom de domaine + son sous-domaine en « www » (Exemple : `domain.tld` et `www.domain.tld`) ;
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
>
> - un seul nom de domaine + son sous-domaine en « www » (exemple : `domain.tld` et `www.domain.tld`) ;
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+> - un seul sous-domaine (Exemple : `sub.domain.tld`).
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> - un seul sous-domaine (exemple : `sub.domain.tld`).
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+>
+> Cela signifie que si vous avez d'autres noms de domaine/sous-domaines déclarés en multisite sur votre hébergement web, ces derniers ne pourront pas bénéficier d'un certificat SSL.
+>
+> Effectivement, il n'est possible d'installer qu'un seul certificat SSL par hébergement web.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> En effet, il n'est possible d'installer qu'un seul certificat SSL par hébergement web.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+> Si vous avez besoin d'activer un certificat SSL pour plusieurs noms de domaine/sous-domaines déclarés sur votre hébergement web, privilégiez l'installation d'un [certificat SSL gratuit Let's Encrypt](/links/web/hosting-options-ssl) ou installez votre propre [certificat SSL personnalisé](/pages/web_cloud/web_hosting/ssl_custom).
+
+**Avant de commander le certificat SSL Sectigo DV sur votre hébergement web**, vérifiez que **le nom de domaine/sous-domaine** concerné par votre certificat SSL : 
+
+- pointe vers l'adresse IP de votre hébergement web ; 
+- est déclaré en multisite sur votre hébergement web.
+
+> [!primary]
+>
+> Dans le cas où vous souhaitez souscrire à un certificat SSL Sectigo DV pour un nom de domaine (Exemple : `domain.tld`), vérifiez bien que son sous-domaine en « www » (Exemple : `www.domain.tld`) pointe bien également vers l'adresse IP de votre hébergement web et est correctement déclaré en multisite.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> Dans le cas où vous souhaitez souscrire à un certificat SSL Sectigo DV pour un nom de domaine (exemple : `domain.tld`), vérifiez bien que son sous-domaine en « www » (exemple : `www.domain.tld`) pointe bien également vers l'adresse IP de votre hébergement web et est correctement déclaré en multisite.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+>
+> En effet, le cas échéant et si vous commandez le certificat SSL Sectigo DV sans vous en assurer, vous devrez corriger cela à postériori. Vous devrez alors supprimer le certificat SSL Sectigo DV précédemment souscrit **sans être remboursé**, puis en commander un nouveau. Ceci afin que le nouveau certificat SSL Sectigo DV englobe bien à la fois votre nom de domaine `domain.tld` et son son sous-domaine en « www » `www.domain.tld`.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> En effet, le cas échéant et si vous commandez le certificat SSL Sectigo DV sans vous en assurer, vous devrez effectuer un correctif a posteriori. Vous devrez alors supprimer le certificat SSL Sectigo DV précédemment souscrit **sans être remboursé**, puis en commander un nouveau. L'objectif est que le nouveau certificat SSL Sectigo DV englobe bien à la fois votre nom de domaine `domain.tld` et son son sous-domaine en « www » `www.domain.tld`.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+>
+> Pour rappel, si vous souscrivez à un certificat SSL Sectigo DV directement pour un sous-domaine (Exemple : `sub.domain.tld`), vous n'êtes pas concerné par cette situation.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> Pour rappel, si vous souscrivez à un certificat SSL Sectigo DV directement pour un sous-domaine (exemple : `sub.domain.tld`), vous n'êtes pas concerné par cette situation.
```

File: pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+Dans la nouvelle fenêtre qui s'affiche, cliquez sur `Valider`{.action} pour être redirigé vers le bon de commande de votre certificat SSL Sectigo DV.
+
+Poursuivez la commande jusqu'au paiement afin de valider la demande de création du certificat SSL Sectigo DV pour votre nom de domaine/sous-domaine sur votre hébergement web.
+
+> [!alert]
+>
+> Une fois la commande validée, la demande de certificat SSL Sectigo DV est envoyée à l'autorité de certification Sectigo.
+>
+> Dès lors, aucun remboursement du SSL Sectigo DV ne sera possible.
```
Comment (Y0Coss) [inline pages/web_cloud/web_hosting/ssl_dv/guide.fr-fr.md]:
```suggestion
> Dès lors, aucun remboursement du SSL Sectigo DV n'est possible.
```

**Pull request merged**